maintaining-and-tuning/monitoring.md

```diff
@@ -153,3 +153,25 @@ access to it can read aggregated metrics about your GRR installation. With these
 about the number of workers, flow activity, and service health can be derived. Make sure to limit
 access to the port, for example by employing a firewall. Furthermore, read
 [Prometheus Security](https://prometheus.io/docs/operating/security/).
+
+## Example visualization setup
+This example will walk you through setting up Grafana as a dedicated visualization software to parse, display, query and set up alerts for the metrics scraped by Prometheus. These instructions assume that GRR server and Prometheus are both up and running.
```

Comment:
The phrase "parse, display, query and set up alerts" sounds like all the verbs correspond to "alerts", as in "parse alerts, display alerts". Maybe better to rephrase it as: "Parse, display and query monitoring metrics and to set up alerts" or something like this.

Reply:
I will rephrase it, and leave the alerting part for another commit/PR since this section is initializing an example visualization setup.

Reply:
After adding sample alerts to the dashboards in [this PR](https://github.com/google/grr/pull/777) for google/grr, I think adding instructions to add alerting here will be more beneficial, as it is quicker and simpler.